KomaMRICore/test/runtests.jl

```diff
@@ -385,3 +386,20 @@ end
     # For the time being, always pass the test
     @test true
 end
+
+@testitem "GPU Functions" tags=[:core] begin
+    using Suppressor
+    import KernelAbstractions as KA
+    include("initialize.jl")
+
+    x = ones(Float32, 1000)
+    if USE_GPU
+        x = x |> gpu
+        @test KA.get_backend(x) isa KA.GPU
+    else
+        @test true
+    end
```

Comment:
I think it would be good to check that

Use_gpu = true
- cpu(gpu(x)) is a CPU array
Use_gpu = false
- cpu and gpu are no-ops